helm/cluster-vsphere/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+annotations:
+  application.giantswarm.io/team: "rocket"
```

Comment:
I did go back and forth on this, but I think ultimately it's Rocket because we're the ones with the knowledge about the underlying components which this chart deploys. Happy to change it though